tasks/eth/029-holocene-system-config-upgrade-and-init-base/.env

```diff
@@ -0,0 +1,6 @@
+ETH_RPC_URL="https://c3-chainproxy-eth-base-mainnet-full-dev.cbhq.net"
+COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1 # This is Base multisig
+FOUNDATION_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A # This is OP multisig
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
```

Comment:
Similar to [task 028](https://github.com/ethereum-optimism/superchain-ops/blob/main/tasks/eth/028-holocene-system-config-upgrade-and-init-multi-chain/.env), can we use safe nonces in this `.env` file? 

For example, I the safe nonces here should be: 
```
SAFE_NONCE_0X7BB41C3008B3F03FE483B28B8DB90E19CF07595C=5
SAFE_NONCE_0X9855054731540A48B28990B63DCF4F33D8AE46A1=17
SAFE_NONCE_0X9BA6E03D8B90DE867373DB8CF1A58D2F7F006B3A=100 
```

You can check for yourself here:
- https://etherscan.io/address/0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c#readProxyContract
- https://etherscan.io/address/0x9855054731540A48B28990B63DCF4F33D8AE46A1#readProxyContract
- https://etherscan.io/address/0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A#readProxyContract